docs/csaf_checker.md

```diff
@@ -6,6 +6,7 @@
 Usage:
   csaf_checker [OPTIONS] domain...
 
+Application Options:
```

Comment:
@JanHoefelmeyer adding trailing newlines should be avoided. (We can address this in a cleanup pr.)